### ISSUE TITLE
About finetune_coco config 

### ISSUE THREAD

**Issue by wanlipeng** [AILab-CVC/YOLO-World#391]
Hello, I have got the **rep_conv.pth, Could you please provide  a complete parameter configuration about “yolo-world-v2-s-rep" without mask-refine, I'm not sure how to choose mosaic_affine_transform、train_pipeline、coco_train_dataset...
